ncm-network/src/main/pan/components/network/core-schema.pan

```diff
@@ -418,6 +418,8 @@ type structure_network = {
     "set_hwaddr" ? boolean
     "nmcontrolled" ? boolean
     "allow_nm" ? boolean
+    @{let Networkmanager manage the dns (nmstate). Default is false}
```

Comment:
with the explicit schema default, you can change it and make it clearer to
`let NetworkManager manage the dns (only for nmstate)`